### PR TITLE
Move out a couple account ids

### PR DESCRIPTION
**Pull request by sastels**
# Summary | Résumé

Move a few more account ids into variables rather than being hard coded.

Added `scan_files_account_id`, `new_relic_aws_account_id`, and `sentinel_sre_aws_account_id` to 1Password for dev / staging / production.

`STAGING_ACCOUNT_ID` and `DEV_ACCOUNT_ID` and already in out GitHub serets.

## Related Issues | Cartes liées

* https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/410

## Before merging this PR

Read code suggestions left by the
[cds-ai-codereviewer](https://github.com/cds-snc/cds-ai-codereviewer/) bot. Address
valid suggestions and shortly write down reasons to not address others. To help
with the classification of the comments, please use these reactions on each of the
comments made by the AI review:

| Classification      | Reaction | Emoticon |
|---------------------|----------|----------|
| Useful              | +1       | 👍        |
| Noisy               | eyes     | 👀        |
| Hallucination       | confused | 😕        |
| Wrong but teachable | rocket   | 🚀        |
| Wrong and incorrect | -1       | 👎        |

The classifications will be extracted and summarized into an analysis of how helpful
or not the AI code review really is.

## Test instructions | Instructions pour tester la modification

Make sure the plans and applies pass!

## Release Instructions | Instructions pour le déploiement

Make sure the plans and applies pass!

## Reviewer checklist | Liste de vérification du réviseur

* [ ] This PR does not break existing functionality.
* [ ] This PR does not violate GCNotify's privacy policies.
* [ ] This PR does not raise new security concerns. Refer to our GC Notify Risk Register document on our Google drive.
* [ ] This PR does not significantly alter performance.
* [ ] Additional required documentation resulting of these changes is covered (such as the README, setup instructions, a related ADR or the technical documentation).

> ⚠ If boxes cannot be checked off before merging the PR, they should be moved to the "Release Instructions" section with appropriate steps required to verify before release. For example, changes to celery code may require tests on staging to verify that performance has not been affected.

### CODE REVIEW

File: aws/common/file_scanning.tf [hunk 1]
```diff
@@ -1,13 +1,9 @@
-locals {
-  scan_files_account = var.env == "production" ? "806545929748" : "127893201980"
-}
-
 module "s3_scan_objects" {
   source = "github.com/cds-snc/terraform-modules//S3_scan_object?ref=v6.1.5"
```
[the remark this answers is not in the document]
Comment (sastels) [reply aws/common/file_scanning.tf:2]: It's in `env/terragrunt.hcl` with the rest of the variables.

File: aws/eks/sentinel.tf [hunk 2]
```diff
@@ -12,7 +12,7 @@ module "sentinel_forwarder" {
   function_name     = "sentinel-cloud-watch-forwarder"
   billing_tag_value = "notification-canada-ca-${var.env}"
 
-  layer_arn = "arn:aws:lambda:ca-central-1:283582579564:layer:aws-sentinel-connector-layer:${var.sentinel_layer_version}"
+  layer_arn = "arn:aws:lambda:ca-central-1:${var.sentinel_sre_aws_account_id}:layer:aws-sentinel-connector-layer:${var.sentinel_layer_version}"
```
[the remark this answers is not in the document]
Comment (sastels) [reply aws/eks/sentinel.tf:15]: it's there...

File: aws/lambda-api/ecr_user.tf [hunk 3]
```diff
@@ -70,6 +70,8 @@ data "aws_iam_policy_document" "ecr" {
     actions = [
       "lambda:GetLayerVersion"
     ]
+    # New Relic Python Lambda layers
+    # https://layers.newrelic-external.com/
     resources = ["arn:aws:lambda:ca-central-1:451483290750:layer:NewRelicPython*:*"]
```
[the remark this answers is not in the document]
Comment (sastels) [reply aws/lambda-api/ecr_user.tf:75]: Generally this is a good idea (and the point of this PR) but This is New Relic's account and not really something for us to configure? Anyways, I thought I'd leave this one here.

File: aws/newrelic/aws_integration.tf [hunk 4]
```diff
@@ -7,7 +7,7 @@ data "aws_iam_policy_document" "newrelic_assume_policy" {
     principals {
       type = "AWS"
       // This is the unique identifier for New Relic account on AWS, there is no need to change this
-      identifiers = [754728514883]
+      identifiers = [var.new_relic_aws_account_id]
```
[the remark this answers is not in the document]
Comment (sastels) [reply aws/newrelic/aws_integration.tf:10]: It's in env/terragrunt.hcl with the rest of the variables.

**Comment by P0NDER0SA** [comment]
Easy right @sastels ?? :D

**Review by P0NDER0SA — APPROVED** [review]
Yes!

**Pull request merged**